tests/test_writers.py

```diff
@@ -191,3 +192,69 @@ def test_write_sssom_ontoportal_json(self):
             self.mapping_count,
             f"{path} has the wrong number of mappings.",
         )
+
+    # TODO: Determine assertions that should pass
+    # TODO: Fix failing assertions that should pass
+    # TODO: before merging, remove pass/fail comments at end of assertion lines
+    def test_missing_entries(self):
```

Comment:
## Get assertions to pass
- [ ] 1. Determine assertions that should pass @matentzn 
- [ ] 2. Fix failing assertions that should pass
- [ ] 3. before merging, remove pass/fail comments at end of assertion lines